pages/common/rscript.md

```diff
@@ -0,0 +1,20 @@
+# R
+
+> Run a script with the R programming language.
+> More information: <https://www.r-project.org>.
+
+- Run an R script:
+
+`Rscript {{file.R}}`
+
+- Start an Rscript in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
```

Comment:
Could you please decide to call it `R script` or `Rscript`, or is there any difference?

Reply:
Sorry for the confusion, I have made changes to clarify this.